routers/openbgpd.php

```diff
@@ -74,6 +74,11 @@ protected function build_as($parameter) {
 
     return array($cmd);
   }
+
```

Comment:
Not required if the IPv6 specific AS path regexp function is removed and its logic included in the existing AS path function.